cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -57,7 +60,7 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 
 // Ethereum Bridge
 parameter_types! {
-	pub storage EthereumGatewayAddress: H160 = H160(hex_literal::hex!("EDa338E4dC46038493b885327842fD3E301CaB39"));
+	pub storage EthereumGatewayAddress: H160 = H160(hex_literal::hex!("b8ea8cb425d85536b158d661da1ef0895bb92f1d"));
```

Comment:
does the new gateway contract support both V1 and V2?

Reply:
Yeah, it supports both. Full API compatibility for V1 is preserved.